pkg/models/job_test.go

```diff
@@ -0,0 +1,144 @@
+//go:build unit || !integration
+
+package models_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+	"github.com/bacalhau-project/bacalhau/pkg/test/mock"
+)
+
+type JobTestSuite struct {
```

Comment:
We should be explicit with the job type we set since there is different behavior regarding jobIDs dependent on its type: https://github.com/bacalhau-project/bacalhau/blob/a78a88dca6d9a4b5977f47a08495b5287f600905/pkg/models/job.go#L295

Reply:
As we are both working on very different timezones, my recommendation is to have bias for action and not block PRs for things not related to the PR itself. Feel free to add comments and I'll make sure I'll address what I can or open follow-up issues to track the ones that I can't, but give approval if there are no concerns with the main scope of the PR.

Added tests in https://github.com/bacalhau-project/bacalhau/pull/4005/commits/d9191d699b5c75a605f7e6d05e13feb9d2fb2fc7